packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -286,8 +295,11 @@ $side-padding: 1.2em;
         &-title {
             text-align: left;
             color: var(--text-less-prominent);
-            font-size: 1.4rem;
-            margin-bottom: 9px;
+            font-size: 1.6rem;
+            margin: auto;
+            &-dark {
+                color: #6e6e6e;
```

Comment:
We avoid declaring colors like that. Try to add a variable for this and put it into theme if required.

Reply:
forgot to remove this, will remove this one

Reply:
Please, review your changes as well to avoid pushing temporary codes :-)